cli/README.md

```diff
@@ -0,0 +1,52 @@
+# Cobalt CLI
+
+Currently this only contains the start of the project structure, a test concept, Makefile and Readme. From a terminal, cd into the `cli` folder and run:
+
+```make deps```
```

Comment:
Is there any reason in particular that we should avoid having a single target which executes these in-order? As a new user I'd imagine that `make build` would do this all on my behalf.

Reply:
I can create an "all" flag. Standard convention to break them out... good catch.